lib/wallaroo/core/windows/panes_range_windows.pony

```diff
@@ -161,12 +163,13 @@ class _PanesSlidingWindows[In: Any val, Out: Any val, Acc: State ref] is
   =>
     let outs = recover iso Array[(Out, U64)] end
     var output_watermark_ts: U64 = 0
+    let range_delay = _range + _delay
```

Comment:
`range_delay` doesn't describe a real concept so I think it's confusing here.  This could be called `trigger_difference`, though it only applies to the difference from the start of the window.

Reply:
Yeah, I was reading through this now and indeed it doesn't make sense. I was optimizing for fitting line 183 on one line;)